mautic/mautic
# 3.3

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

Added Canton Jura to Swiss regions.


#### Description:
Canton Jura was missing.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Open a contact, Edit and see the State/Region/Canton dropdown
3. see Canton Jura in the dropdown
